packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -25,6 +25,7 @@ import {
 } from '@automattic/composite-checkout';
 import formatCurrency from '@automattic/format-currency';
 import styled from '@emotion/styled';
+import { useViewportMatch } from '@wordpress/compose';
```

Comment:
To use this properly you'll need to add `@wordpress/compose` to the package.json dependencies for `wpcom-checkout`.

Reply:
Done